### PR TITLE
fix session-screenshot aws lambda

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

Our backend uses session-screenshot lambda to attach session screenshots to slack comments.
This PR fixes the AWS lambda (hosted at https://ygh5bj5f646ix4pixknhvysrje0haeoi.lambda-url.us-east-2.on.aws/session-screenshots?project=1&session=238920399&ts=165876&chunk=2).

## How did you test this change?

Locally deploying and invoking lambda. `main` push CD will auto-deploy lambda.
Screenshot 2023-06-12 at 4 19 32 PM

## Are there any deployment considerations?

CD will redeploy AWS lambda.

### CODE REVIEW

**Comment by Vadman97** [comment]
whoops, wrong pr

File: render/package.json [hunk 1]
```diff
@@ -2,78 +2,29 @@
...
 	"version": "0.0.0",
+	"type": "module",
+	"main": "./dist/index.js",
+	"module": "./dist/index.js",
 	"scripts": {
-		"build": "tsc --build ./",
-		"single": "DEV=true ts-node -r tsconfig-paths/register ./src",
-		"dev": "DEV=true nodemon",
-		"publish": "zip -r function.zip dist node_modules -x node_modules/puppeteer/.local-chromium/\\* && aws s3 cp function.zip s3://highlight-lambda-code/session-screenshots.zip && rm function.zip && aws lambda update-function-code --function-name session-screenshots --s3-bucket highlight-lambda-code --s3-key session-screenshots.zip"
+		"build": "tsup",
```
Comment (SpennyNDaJets) [inline render/package.json:9]: 👍

**Review by SpennyNDaJets — APPROVED** [review]
Code looks good to me!

Wasn't able to test locally, but test plan makes sense to me

File: render/src/render.ts [hunk 2]
```diff
@@ -35,24 +36,27 @@ export async function render(
 		throw new Error('timestamp or fps must be provided')
 	}
 	events = events.replace(/\\/g, '\\\\')
+	console.log('events', { events })
```
[the remark this answers is not in the document]
Comment (Vadman97) [reply render/src/render.ts:39]: they don't hurt - they go to cloudwatch where they automatically expire

**Pull request merged**